include/Dollar.h

```diff
@@ -2,10 +2,12 @@
 #ifndef DOLLAR_H                                                                                              
 #define DOLLAR_H                                                                                              
                                                                                                               
-class Dollar                                                                                                  
+class Dollar                                                                                              
 {
-public:                                                                                                          
-Dollar();                                                                                                 
+public:
+int amount;                                                                                   
```

Comment:
Please leave either 4 space or a tab before `int amount` and the other members or methods.